bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_fi.properties

```diff
@@ -79,6 +79,13 @@ channel-group-type.netatmo.signal.label = Signaali
 channel-group-type.netatmo.siren.label = Sireenin tila
 channel-group-type.netatmo.status-doorbell.label = Kameran tila
 channel-group-type.netatmo.status.label = Kameran tila
+channel-group-type.netatmo.sub-event.label = Tapahtuman alalaji
+channel-group-type.netatmo.sub-event.channel.time.label = Tapahtuman alalajin aikaleima
+channel-group-type.netatmo.sub-event.channel.time.description = Hetki, jona tapahtuman alalaji tapahtui.
+channel-group-type.netatmo.sub-event.channel.vignette.label = Vinjetti
+channel-group-type.netatmo.sub-event.channel.vignette.description = Tilannekuvan vinjetti.
+channel-group-type.netatmo.sub-event.channel.vignette-url.label = Vinjetin URL
+channel-group-type.netatmo.sub-event.channel.vignette-url.description = Vinjetin URL-osoite.
```

Comment:
Please do not update each translation in your PR.
It has to be done using Crowdin (after the PR is merged).

Reply:
it was just copy and paste needed, but ok, I have reverted it